docs/3.0.0-alpha.x/guides/deployment.md

```diff
@@ -371,16 +370,23 @@ npm install pg
 }
 ```
 
-3. Push your local changes to your project's GitHub repository.
+3. Install the **Strapi Provider Upload AWS S3 Plugin**: `Path: ./my-project/`. This plugin will allow configurations for each active environment. **NOTE:** You can only update the settings for _production_ when your Strapi project is actually in production.
 
 ```bash
-git commit -am 'installed pg and update production/database.json file'
+npm install strapi-provider-upload-aws-s3
```

Comment:
`strapi-provider-upload-aws-s3@beta` as the "latest" version is alpha.15

Reply:
@derrickmehaffy this is the Alpha version of the docs.  At some point the latest version of the plugin won't work with the alpha version of Strapi.  Therefore, I will leave it like this, and allow the user to determine which version.

Reply:
Clarified in Beta docs.

Reply:
@davidkartuzinski should still use @alpha in the alpha docs

Reply:
![image](https://user-images.githubusercontent.com/8593673/59674809-df4d5100-9178-11e9-9c6d-a51408e73a1f.png)

Reply:
great catch. Thanks.